source/docs/3/importing_spreadsheets_csv.rst

```diff
@@ -5,44 +5,65 @@ Many times the GIS data comes in a table or an Excel spreadsheet. Also, if you h
 Overview of the task
 --------------------
 
-We will be importing a text file of earthquake data to QGIS. 
+We will be importing a text file (tab seperated) of earthquake data to QGIS. 
 
 Get the data
 ------------
 
-NOAA’s National Geophysical Data Center produces a great dataset of all significant earthquakes since 2150 BC. `Learn more. <http://www.ngdc.noaa.gov/nndc/struts/form?t=101650&s=1&d=1>`_
+1. NOAA’s National Geophysical Data Center produces a great dataset of all significant earthquakes since 2150 BC. `Visit the NOAA NCEI portal <https://www.ngdc.noaa.gov/hazel/view/hazards/earthquake/search>`_ and enter :guilabel:`Min` as ``1900`` and :guilabel:`Max` as ``2000``.  This will return all earthquake incidents that occurred and were recorded by NOAA between those years. For other specific results, you can filter with different parameters. Click :guilabel:`Search`.
 
-Download `Significant Earthquake Database <http://www.ngdc.noaa.gov/nndc/struts/results?type_0=Exact&query_0=$ID&t=101650&s=13&d=189&dfn=signif.txt>`_ text file.
+  .. image:: /static/3/importing_spreadsheets_csv/images/data01.png
+     :align: center
+
+2. As a result, we got *2585* earthquake incidence. Click on the :guilabel:`Download TSV` icon. 
+
+  .. image:: /static/3/importing_spreadsheets_csv/images/data02.png
+     :align: center
 
-For convenience, you may directly download a copy of both the datasets from the links below:
+For convenience, you may directly download a copy of the above datasets from the link below:
 
-`signif.txt <https://www.qgistutorials.com/downloads/signif.txt>`_ 
+`earthquakes_2021_11_25_14_31_59_0530 <https://www.qgistutorials.com/downloads/earthquakes-2021-11-25_14-31-59_+0530.tsv>`_ 
 
 Data Source [NGDC]_
 
 Procedure
 ---------
 
-1. Examine your tabular data source. To import this data to QGIS, you will have to save it as a text file and need at least 2 columns which contain the X and Y coordinates. If you have a spreadsheet, use `Save As` function in your program to save it as a `Tab Delimited File` or a `Comma Separated Values (CSV)` file. Once you have the data exported this way, you can open it in a text editor such as Notepad to view the contents. In case of the Significant Earthquake Database, the data already comes as a text file which contains latitude and longitude of the earthquake centers along with other related attributes. You will see that each field is separated by a TAB. 
+1. Examine your tabular data source. The downloaded earthquake database contains the ``Latitude`` and ``Longitude`` of the earthquake centers and other related attributes. To import this data to QGIS, you need at least two columns that contain the X and Y coordinates. Open the data in a text editor such as Notepad/TextMate to view the contents. You will see that a TAB separates each field.
```

Comment:
Add back the details about how to export spreadsheet to CSV.

"contains the ``Latitude`` and ``Longitude`` 
to
contains the ``Latitude`` and ``Longitude``  fields indicating the location of the earthquake centers.

Instead of "To import this data to QGIS, you need at least two columns that contain the X and Y coordinates."

"We will use these fields to import the file as a point layer."

We will

Reply:
"If you have a spreadsheet, use `Save As` function in your program to save it as a `Tab Delimited File` or a `Comma Separated Values (CSV)` file."

Shall I add this as a note?

Reply:
yes